apps-client/mobile/components/common/header-navigation.tsx

```diff
@@ -0,0 +1,37 @@
+import { View } from 'react-native'
+
+import { Button } from '@pple-today/ui/button'
+import { Icon } from '@pple-today/ui/icon'
+import { H1 } from '@pple-today/ui/typography'
+import { useRouter } from 'expo-router'
+import { ArrowLeftIcon, LucideIcon } from 'lucide-react-native'
+
+interface HeaderProps {
+  icon: LucideIcon
+  title: string
+  extraFunc?: () => void
+}
+
+export function HeaderWithNoDescription(props: HeaderProps) {
```

Comment:
normally i dont suggest extracting component like this
but this is fine if done correctly

can you rename this to be more broad like just `Header` or `PageHeader`